.github/workflows/create-release-candidate.yml

```diff
@@ -0,0 +1,23 @@
+name: Create release candidate 
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-locked:
+        type: boolean
+        description: Uncheck if you want to follow SemVer versioning rather than version-lock to axe-core.
```

Comment:
This isn't a great way to put this. When updating axe-core to a minor or major version it would be still version-locked to release as a major version but the release-candidate script itself would prevent that. Instead I propose this as the wording (@dequejenn should approve to):

> Should this action be allowed to create a minor or major release?

With the default being the opposite value of the `version-locked` setting of the repo.

Reply:
@straker - what Gabe and I were realizing is if we had the checkbox be `checked=version locked` and `unchecked=not version locked/semver` then there wasn't additional code to map the input field. I am fine with changing the text but the test should lead someone to check the box only if we are NOT following semver.

Reply:
@Zidious - Steve came up with the best one. Please change wording on teh checkbox to  `Prevent this action from creating a minor or major release`

Reply:
So the input will still be for the `version-locked` variable, it's just the wording change. For the desire to have check the checkbox box === not following semver then we can use the wording:

> Prevent this action from creating a minor or major release

Reply:
Tested visuals in test-repo: 

<img width="413" alt="image" src="https://github.com/dequelabs/axe-core-npm/assets/41127686/d64a1e6f-dad0-4e96-b213-61b26f0bdc06">